src/rust/ensogl/lib/core/src/display/shape/text/text_field/render.rs

```diff
@@ -136,10 +137,17 @@ impl TextFieldSprites {
     }
 
     fn create_selection_system<'t,S:Into<&'t Scene>>(scene:S) -> ShapeSystem {
-        const ROUNDING:f32       = 3.0;
-        let width                = "input_size.x";
-        let height               = "input_size.y";
-        let selection_definition = shape::Rect((width,height));
+        const ROUNDING:f32           = 3.0;
+        let width                    = "input_size.x";
+        let height                   = "input_size.y";
+        let selection_definition     = shape::Rect((width,height));
+        let scene                    = scene.into();
+        let styles                   = StyleWatch::new(&scene.style_sheet);
+        let selection_color_path     = "text.selection.color";
+        let selection_color_fallback = color::Lcha::new(0.7,0.0,0.125,0.7);
```

Comment:
Also, I dont like this. This forces us to define a lot of hardcodded colors and variables across different files. Instead, default always to a default color - let it be red or black. There is an assumption, that there should also be loaded a "default style", which user can override and specify better. This is why choosing a style consumes a list like `app.themes.set_enabled(&["dark"]);` - this allows you to enable multiple styles and if a variable was not found in the first one, it will be searched in the second etc. So instead of hardcoding such colors, lets always assume the default style will be loaded, and instead of suing function `get_color_or`, write a function `get_color` which defaults to red.